backend/src/models/farmer.ts

```diff
@@ -13,6 +13,6 @@ const farmerSchema = new Schema<IFarmer>({
 });
 
 // 3. Create a Model.
-const Farmer: Model<IFarmer> = model<IFarmer>("farmers", farmerSchema);
+const Farmer: Model<IFarmer> = model<IFarmer>("Farmer", farmerSchema);
```

Comment:
Hey Justin, I think this needs to stay "farmers" because it is the name of the collection and it is usually plural